hack/build-jsonnet.sh

```diff
@@ -22,15 +22,23 @@ while IFS= read -r line; do
 	files+=("$line")
 done < <(jq -r 'keys[]' "${TMP}/main.json")
 
-for file in "${files[@]}"
-do
-	dir=$(dirname "${file}")
-	path="${prefix}/${dir}"
-	mkdir -p "${path}"
-	# convert file name from camelCase to snake-case
-	fullfile=$(echo "${file}" | sed 's/\(.\)\([A-Z]\)/\1-\2/g' | tr '[:upper:]' '[:lower:]')
-	jq -r ".[\"${file}\"]" "${TMP}/main.json" | gojsontoyaml > "${prefix}/${fullfile}.yaml"
+# process all files in parallel
+for file in "${files[@]}"; do
+	(
+		dir=$(dirname "${file}")
+		path="${prefix}/${dir}"
+		mkdir -p "${path}"
+
+		# convert file name from camelCase to snake-case
+		fullfile=$(echo "${file}" | sed 's/\(.\)\([A-Z]\)/\1-\2/g' | tr '[:upper:]' '[:lower:]')
+		jq -r ".[\"${file}\"]" "${TMP}/main.json" | gojsontoyaml > "${prefix}/${fullfile}.yaml"
+	)&
```

Comment:
```suggestion
for file in "${files[@]}"; do
		dir=$(dirname "${file}")
		path="${prefix}/${dir}"
		mkdir -p "${path}"

		# convert file name from camelCase to snake-case
		fullfile=$(echo "${file}" | sed 's/\(.\)\([A-Z]\)/\1-\2/g' | tr '[:upper:]' '[:lower:]')
		jq -r ".[\"${file}\"]" "${TMP}/main.json" | gojsontoyaml > "${prefix}/${fullfile}.yaml" &
```
I get slightly better results with this change (shave a few seconds off the results fairly consistently). Can you give this a try as well?

Reply:
This was my first version :) but I felt starting a subshell at the start of for loop makes it easier to reason with. Since it boils down to 
```
for f in files {
    ( process this file) &  // in background
    wait if there are more than N processes
}
```